### ISSUE TITLE
Could we have `status/pending-qa-verification` label ?

### ISSUE THREAD

**Issue by ibuziuk** [openshiftio/openshift.io#2099]
When the fix is promoted to prod / prod-preview it would be very nice to have a possibility to set `status/pending-qa-verification` label which would notify that QA verification is required. For now there is labels “verified-fix-in-prod-preview” / “verified-fix-in-prod” but nothing to notify QA team about verification request.

Wiki description:

> | `status/pending-qa-verification` | This issue is pending QA verification. Once verification is done “verified-fix-in-prod-preview” / “verified-fix-in-prod” label should be set and the issue could be closed

**Comment by bartoszmajsak** [comment]
No way of having automated verification before any fix lands in prod-preview?

**Comment by joshuawilson** [comment]
Labels do NOT notify when set. But sure you can have that label.

**Comment by ibuziuk** [comment]
at very least for che there is no such verification in place (not sure about if other component are fully automated and require no manual tests these days), but @rhopp is working on having this done at least partially [1]. Also there are cross-component & prod-preview specific issues like [2] where QA verification would be nice to have. Currently in order to get QA attention we need to ping them in github issue directly. Having the label would allow QA to see where some involvement from their side is required

[1] https://github.com/redhat-developer/rh-che/issues/447
[2] https://github.com/openshiftio/openshift.io/issues/2063

**Comment by joshuawilson** [comment]
We don't have `verified-fix-in-prod-preview` OR `verified-fix-in-prod`

**Comment by ibuziuk** [comment]
> Labels do NOT notify when set. But sure you can have that label.

notify = QA will be sorting issues by this label on daily basis and see where their involvement is required

> We don't have verified-fix-in-prod-preview OR verified-fix-in-prod
We do. This is not documented in wiki though. Do not know who os the creator

**Comment by joshuawilson** [comment]
I missed them at the bottom, i thought they would have been under status.
I added the requested label.

**Comment by ibuziuk** [comment]
@joshuawilson thanks!
